src/Controllers/MovieController.cs

```diff
@@ -89,21 +87,47 @@ public IActionResult EditMovie(MovieViewModel movie, int id)
 
         [HttpGet("")]
         [HttpGet("ListMovies/{searchString?}")]
```

Comment:
I don't know if this is a rebase issue, but don't forget to remove the searchstring from the URL